Fails to upload

First of all: thanks a lot for this project! :)
Unfortunately it fails to upload for me (see https://github.com/LinusU/wext-shipit/issues/10, too):

```
•100% ➜ npm run shipit:opera

> amazon-tag-remover@0.5.2 shipit:opera /Users/timbru/work/private/amazon-tag-remover
> shipit opera dist

✖ Error: Node is either not visible or not an HTMLElement
Error: Node is either not visible or not an HTMLElement
    at ElementHandle._clickablePoint (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/JSHandle.js:199:13)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async ElementHandle.click (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/JSHandle.js:265:20)
    at async DOMWorld.click (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/DOMWorld.js:367:5)
    at async uploadOperaExtension (/Users/timbru/work/private/amazon-tag-remover/node_modules/upload-opera-extension/index.js:14:5)
    at async module.exports (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/lib/opera.js:43:5)
    at async main (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/bin.js:31:5)
  -- ASYNC --
    at ElementHandle.<anonymous> (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/helper.js:111:15)
    at DOMWorld.click (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/DOMWorld.js:367:18)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async uploadOperaExtension (/Users/timbru/work/private/amazon-tag-remover/node_modules/upload-opera-extension/index.js:14:5)
    at async module.exports (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/lib/opera.js:43:5)
    at async main (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/bin.js:31:5)
  -- ASYNC --
    at Frame.<anonymous> (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/helper.js:111:15)
    at Page.click (/Users/timbru/work/private/amazon-tag-remover/node_modules/puppeteer/lib/Page.js:986:29)
    at uploadOperaExtension (/Users/timbru/work/private/amazon-tag-remover/node_modules/upload-opera-extension/index.js:14:16)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async module.exports (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/lib/opera.js:43:5)
    at async main (/Users/timbru/work/private/amazon-tag-remover/node_modules/@wext/shipit/bin.js:31:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amazon-tag-remover@0.5.2 shipit:opera: `shipit opera dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amazon-tag-remover@0.5.2 shipit:opera script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/timbru/.npm/_logs/2019-07-18T11_47_06_431Z-debug.log
```

Let me know if I can assist further in debugging this.

--- a/index.js
+++ b/index.js
@@ -7,20 +7,13 @@ async function uploadOperaExtension (options) {
   const page = await browser.newPage()
 
   try {
-    // Go to main page
-    await page.goto('https://addons.opera.com')
-
-    // Go to login page
-    await page.click('a#login')
+    // Directly go to the extension page - the browser will be redirected to the auth page and back once the login succeeded
+    await page.goto(`https://addons.opera.com/developer/package/${options.extensionId}/`)
 
     // Perform login
     await page.type('input[name=email]', options.email)
     await page.type('input[name=password]', options.password)
     await page.click('button[type=submit]')
-    await page.waitForSelector('div#account')
-
-    // Go to extensions page
-    await page.goto(`https://addons.opera.com/developer/package/${options.extensionId}/`)
 
     // Wait for, and then click, "Versions"
     await page.waitForSelector('ul.nav .uib-tab:nth-child(2) a')
@@ -43,7 +36,7 @@ async function uploadOperaExtension (options) {
       try {
         errorText = await page.evaluate(() => document.querySelector('flash-message div.alert span.ng-scope').innerText)
       } catch (_) {
-        errorText = 'Unknown error occurde while uploading extension'
+        errorText = 'Unknown error occurred while uploading extension'
       }
 
       throw new Error(errorText)

